TS03-Subset-Data-and-No-Data-Values-In-R.Rmd

```diff
@@ -162,119 +168,171 @@ write.csv(harMet15.09.11, file="Met_HARV_15min_2009_2011.csv")
-cause error in calculations.  By default `R` will not calculate certain function
-if they have a NA value in it.  
+### Managing noData Values in Our Data
+
+For this lesson, we are exploring the patterns of precipitation,
+and termperature as they related to green-up and brown-down of vegetation at 
+Harvard Forest. To view overall trends during these early exploration stages, it 
+is ok for us to leave out the `noData` values in our plots. 
+
+
+<i class="fa fa-star"></i> **Data Tip:** If we wanted to perform more advanced 
+statistical analysis, we might consider gap-filling as our next step. Many data 
+products, from towers such as FluxNet include a higher level, already gap-filled
+product that we can download! <a href="http://www.archive.arm.gov/Carbon/gapfilling/gapfilling.html"
+target="_blank">More on Gap Filling</a>
+{: .notice}
+
+
+### noData Values Can Impact Calculations
+
+It is important to consider `noData` values when performing calculations on our
+data. For example, `R` will not properly calculate certain functions if there are 
+`NA` values in the data, unless we explicetly tell it to ignore them. 
+
 ``` {r na-in-calculations}
+
 #calculate mean of air temperature
 mean(harMet15.09.11$airt)
 
 ```
 
-`R` will not return a value for the mean as there NA values in the air 
-temperature.  However, we know that there are only 2 of 105,108 observations of 
-air temperature missing, so we aren't that worried about those two missing values
-skewing the three year air temperature mean.  We can tell `R` to not include the
-missing values by using `na.rm=` (NA.remove) within the function.  
+`R` will not return a value for the mean as there `NA` values in the air 
+temperature column. Because there are only 2 missing values (out of 105,108) for 
+air temperature, we aren't that worried about a skewed 3 year mean. We can tell 
+`R` to ignore missing values in the mean calculations using `na.rm=` (NA.remove).
 
 ``` {r na-rm}
 #calculate mean of air temperature, ignore NA values
-mean(harMet15.09.11$airt, na.rm=TRUE)
+mean(harMet15.09.11$airt, 
+     na.rm=TRUE)
 
 ```
 
-We now see that the average air temperature across the three years was 8.5°C.  
+We now see that the 3-year average air temperature is 8.5°C.  
+
+<div id="challenge" markdown="1">
 
-# Challenge: Import, Understand Metadata, and Clean a Data Set
+## Challenge: Import, Understand Metadata, and Clean a Data Set
 We have been using the 15-minute data from the Harvard Forest. However, overall
 we are interested in larger scale patterns of greening-up and browning-down.  
-Daily atmospheric data are therefore far more appropriate for our work than the 
-15-minute data.  
+Thus a daily summary is sufficient for us to see overall trends.
 
 1. Import the Daily Meteorological data from the Harvard Forest (if you haven't already done so in the [Brief Tabular Time Series lesson ]({{site.baseurl}}/R/Brief-Tabular-Time-Series-qplot/ "for HarMet_Daily"). )
```

Comment:
Lesson have has been changed to:  Intro to Time Series Data in R - Managing Date / Time Format and Conversion and Simple Plots using ggplot2